### PR TITLE
TS: fix Python v2/v3 compatibility

### PR DESCRIPTION
**Pull request by cv3d**
### This pullrequest changes
Fix for the test suite reporting tools to work with Python 2.7

### CODE REVIEW

File: modules/ts/misc/table_formatter.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import sys, re, os.path, cgi, stat, math
 from optparse import OptionParser
 from color import getColorizer, dummyColorizer
+import six
```
Comment (alalek) [inline modules/ts/misc/table_formatter.py]: 'six' package is not a builtin package (not bundled with Python)

Comment (cv3d) [reply modules/ts/misc/table_formatter.py]: fixed!

**Comment by alalek** [comment]
What tool are you fixing here? (please provide some sample)
I'm still observing failures with `summary.py` and Python 3.

**Comment by cv3d** [comment]
I used Python 2.7, but let me check using Python 3

**Comment by cv3d** [comment]
```
>python -V
Python 3.6.3 :: Intel Corporation

>python ..\opencv\modules\ts\misc\run.py -t dnn --gtest_filter=*Conv*/1?
.....

>python ..\opencv\modules\ts\misc\report.py *.xml

dnn_20180929-185848.xml

                                                 Name                                                                          Min
                                                                                                                      outli
conv::Conv::(GFLOPS=0.958, K=[3 x 3], IN={1, 384, 19, 19}, OCN=384, PM=SAME, OCV/CPU)                                           9.38                                          2.21
conv::Conv::(GFLOPS=0.958, K=[3 x 3], IN={1, 384, 19, 19}, OCN=384, PM=SAME, OCV/OCL)                                           5.95    6.38       6.45       6.46         0.29
conv::Conv::(GFLOPS=1.659, K=[3 x 3], IN={1, 960, 10, 10}, OCN=960, PM=SAME, OCV/CPU)                                                                                          1.77
conv::Conv::(GFLOPS=1.659, K=[3 x 3], IN={1, 960, 10, 10}, OCN=960, PM=SAME, OCV/OCL)                                                                                          0.56
conv::Conv::(GFLOPS=1.659, K=[3 x 3], IN={1, 960, 10, 10}, OCN=960, PM=SAME, OCV/OCL_FP16)                                                                                     1.30
conv::Conv::(GFLOPS=2.156, K=[3 x 3], IN={1, 576, 19, 19}, OCN=576, PM=SAME, OCV/CPU)                                                                                          4.31
conv::Conv::(GFLOPS=2.156, K=[3 x 3], IN={1, 576, 19, 19}, OCN=576, PM=SAME, OCV/OCL)                                                                                          2.00
conv::Conv::(GFLOPS=2.156, K=[3 x 3], IN={1, 576, 19, 19}, OCN=576, PM=SAME, OCV/OCL_FP16)                                                                                     0.71
conv::Conv::(GFLOPS=6.641, K=[3 x 3], IN={1, 64, 150, 200}, OCN=192, PM=SAME, BIAS, OCV/OCL)             10                                                                   0.96
conv::Conv::(GFLOPS=6.641, K=[3 x 3], IN={1, 64, 150, 200}, OCN=192, PM=SAME, BIAS, OCV/OCL_FP16)                                                                              3.47

>python ..\opencv\modules\ts\misc\summary.py *.xml

Geometric mean (ms)

                                                 Name

conv::Conv::(GFLOPS=0.958, K=[3 x 3], IN={1, 384, 19, 19}, OCN=384, PM=SAME, OCV/CPU)                  15.69
conv::Conv::(GFLOPS=0.958, K=[3 x 3], IN={1, 384, 19, 19}, OCN=384, PM=SAME, OCV/OCL)
conv::Conv::(GFLOPS=1.659, K=[3 x 3], IN={1, 960, 10, 10}, OCN=960, PM=SAME, OCV/CPU)                  29.17
conv::Conv::(GFLOPS=1.659, K=[3 x 3], IN={1, 960, 10, 10}, OCN=960, PM=SAME, OCV/OCL)                  17.99
conv::Conv::(GFLOPS=1.659, K=[3 x 3], IN={1, 960, 10, 10}, OCN=960, PM=SAME, OCV/OCL_FP16)             13.97
conv::Conv::(GFLOPS=2.156, K=[3 x 3], IN={1, 576, 19, 19}, OCN=576, PM=SAME, OCV/CPU)                  41.09
conv::Conv::(GFLOPS=2.156, K=[3 x 3], IN={1, 576, 19, 19}, OCN=576, PM=SAME, OCV/OCL)                  24.68
conv::Conv::(GFLOPS=2.156, K=[3 x 3], IN={1, 576, 19, 19}, OCN=576, PM=SAME, OCV/OCL_FP16)             15.12
conv::Conv::(GFLOPS=6.641, K=[3 x 3], IN={1, 64, 150, 200}, OCN=192, PM=SAME, BIAS, OCV/OCL)           34.01
conv::Conv::(GFLOPS=6.641, K=[3 x 3], IN={1, 64, 150, 200}, OCN=192, PM=SAME, BIAS, OCV/OCL_FP16)      49.65
```

@alalek Anything else?

**Comment by alalek** [comment]
Please rebase this onto 3.4 branch

**Pull request merged**